.gitignore

```diff
@@ -205,3 +205,22 @@ Chart.lock
 
 # Might be generated when you build wheels
 pip-wheel-metadata
+
+# tmp ...
```

Comment:
Can you say more about it?

Reply:
I think it may be unrelated

Reply:
Ok, I'm back guess, will the `tmp` mean  temporary, but this changes should be just  local on my machine :smile: